lib/rules/selector-max-specificity/README.md

```diff
@@ -109,17 +109,17 @@ The following patterns are _not_ considered problems:
 
 <!-- prettier-ignore -->
 ```css
-:global(.foo) .bar {}
+:is(.foo) .bar {}
 ```
 
 <!-- prettier-ignore -->
 ```css
-:local(.foo.bar) {}
+:has(.foo.bar) {}
 ```
 
 <!-- prettier-ignore -->
 ```css
-:local(.foo, :global(.bar).baz) {}
+:has(.foo, :is(.bar).baz) {}
 ```
```

Comment:
minor
remark

These examples make no sense to me since

> The :has() pseudo-class takes on the specificity of the most specific selector in its arguments the same way as :is() and :not() do.

it should regardless of the `ignoreSelectors`.
Are those even relevant examples?

Reply:
Ah, yes, needs to be something like `:host` which does have its own specificity.